internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -149,15 +148,25 @@ type RuntimeManager interface {
 	PerformComponentDiagnostics(ctx context.Context, additionalMetrics []cproto.AdditionalDiagnosticRequest, req ...component.Component) ([]runtime.ComponentDiagnostic, error)
 }
 
-// OTelManager provides an interface to run and update the runtime.
+// OTelManager provides an interface to run components and plain otel configurations in an otel collector.
 type OTelManager interface {
 	Runner
 
-	// Update updates the current configuration for OTel.
-	Update(cfg *confmap.Conf)
+	// UpdateCollector updates the current plain configuration for the otel collector.
+	UpdateCollector(cfg *confmap.Conf)
 
-	// Watch returns the chanel to watch for configuration changes.
-	Watch() <-chan *status.AggregateStatus
+	// WatchCollector returns a channel to watch for collector status updates.
+	WatchCollector() <-chan *status.AggregateStatus
```

Comment:
thinking out loud here, by renaming and making some of these funcs Collector explicit feels a little bit off from the end-goal we are after in the long run, which is to abstract out any otel collector specific logic and converge our codebase to support both the former and the pre-existing other runtimes. Thus if we keep the funcs of this interface as generic as possible we could always try and migrate this manager next to the other that is for "traditional" beats, etc. Again this comment is not a blocking one and I write it in a thinking out loud fashion 🙂